website/content/docs/upgrade/upgrade-specific.mdx

```diff
@@ -13,12 +13,18 @@ upgrade. However, specific versions of Nomad may have more details provided for
 their upgrades as a result of new features or changed behavior. This page is
 used to document those details separately from the standard upgrade flow.
 
+## Nomad 1.7.2
+
+Nomad 1.7.2 fixes a critical bug in CPU fingerprinting in Nomad 1.7.0 and
+1.7.1. You should not install Nomad 1.7.0 or 1.7.1 and instead install the
+latest Nomad 1.7.x version.
+
```

Comment:
Should we include a note here about ensuring that `dmidecode` is installed on Linux?

Reply:
Well we do mention it already in the upgrade guide as of https://github.com/hashicorp/nomad/pull/19348

Reply:
oh, which I guess never got backported... :disappointed: